VERSIONS

```diff
@@ -1,13 +1,14 @@
 CXX_STANDARD "20"
 STARTER_REV "v0.18.6-preview-4"
 SYNCER_REV "v2.19.7"
-GCC_LINUX "11.2.1_git20220219-r2"
+GCC_LINUX "13.2.0"
```

Comment:
Why are we changing the version of gcc here as well?
I am not sure if that is used anywhere later in the toolchain, but I am pretty sure that compiling 3.11 with g++-13.2.0 will not work.

Reply:
Removed.